releases/2022-11-25-releasenotes.md

```diff
@@ -0,0 +1,154 @@
+---
+draft: false
+title: "Handle errors the GraphQL way and more results to your query"
+description: "Vendia Share updated the response status code and allowing up to 1000 query results"
+date: '2022-11-25'
+authors:
+  - Li Dai
+tags:
+- Improvements
+- Developer Experience
+- GraphQL
+---
+
+Vendia Engineers worked tirelessly to improve our product and ensure that we follow and set best practices for industrial standards. Here are the newest updates!
+
+# Error handling the way it should with [GraphQL](https://graphql.org/). 
+
+Errors are not scary as long as we handle them well. In the past Vendia's end point would return 4XX and 5XX status codes for any http request failures. But that has changed as we decided to follow GraphQL's [recommended approach](https://graphql.github.io/graphql-over-http/draft/#sec-Field-errors-encountered-during-execution) over http calls.
```

Comment:
I think customers would only see 4xx errors, right? 5xx were internal.  Given this, it's probably worth stating the specific types of errors that went from 4xx to 2xx that are noted here: https://www.notion.so/vendia/802a11ba67a74bbc948204b52f49af13?v=b38bad4e3af648db9df16d40d1250ade

Reply:
I'll make a table for this.

Reply:
done